deployment/config.go

```diff
@@ -82,12 +83,26 @@ type ElasticSearchSettings struct {
 
 // IsValid reports whether a given deployment config is valid or not.
 func (c *Config) IsValid() error {
+	if !checkPrefix(c.MattermostDownloadURL) {
+		return fmt.Errorf("mattermost download url is not in correct format: %q", c.MattermostDownloadURL)
+	}
+
+	if !checkPrefix(c.LoadTestDownloadURL) {
+		return fmt.Errorf("load-test download url is not in correct format: %q", c.LoadTestDownloadURL)
+	}
```

Comment:
This raises an interesting question: how much of these kind of checks do we delegate to the `defaults` package?

Reply:
Hehe .. I think this would fall slightly out of scope of the defaults package. Checking a prefix may be too much of an application logic. 0/5 on whatever you guys decide.

Reply:
Maybe we can add multiple validation rules with a logic of `&&` and `||`.

Reply:
I think he was being serious :stuck_out_tongue:

Reply:
I am totally 0/5. All yours @isacikgoz.

Reply:
Created a ticket for this, no need to block this one.